src/org/usfirst/frc/team3786/robot/commands/DriveCommand.java

```diff
@@ -0,0 +1,57 @@
+package org.usfirst.frc.team3786.robot.commands;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.command.Command;
+
+import org.usfirst.frc.team3786.robot.Robot;
+
+/**
+ *
+ */
+public class DriveCommand extends Command {
+	Talon leftMotor, rightMotor;
+	Joystick joystick;
+	public DriveCommand() {
+		// Use requires() here to declare subsystem dependencies
+		requires(Robot.drive);
+	}
+
+	// Called just before this Command runs the first time
+	@Override
+	protected void initialize() {
+		joystick = new Joystick(0);
+		leftMotor = new Talon(0); //change the value to the correct value for the location
+		rightMotor = new Talon(1); //change the value to the correct value for the location
```

Comment:
Instead of creating new Joystick and Talon instances here, have a central object (like Robot) that keeps all of the individual hardware instances.